convene-web/.env.example

```diff
@@ -0,0 +1,2 @@
+DEMO_ENABLED=false
+SYSTEM_TEST=false
```

Comment:
Would we want this to be true on local dev environments so that the system tests can run against the local dev environment?